packages/interface-ipfs-core/src/ping/ping.js

```diff
@@ -49,11 +49,16 @@ module.exports = (common, options) => {
       expect(pongs.length).to.equal(count)
     })
 
-    it('should fail when pinging a peer that is not available', () => {
+    it('should fail when pinging a peer that is not available', async () => {
       const notAvailablePeerId = 'QmUmaEnH1uMmvckMZbh3yShaasvELPW4ZLPWnB4entMTEn'
       const count = 2
 
-      return expect(all(ipfsA.ping(notAvailablePeerId, { count }))).to.eventually.be.rejected()
```

Comment:
Why was this necessary?

Reply:
i was trying to catch an uncaught rejection that happens sometimes, because the new runner fails when these happen now.
this seems to help a bit but 100%